pokemongo_bot/cell_workers/incubate_eggs.py

```diff
@@ -176,6 +180,9 @@ def _check_inventory(self, lookup_ids=[]):
                             pokemon.get('individual_stamina', 0)
                         ]})
                     matched_pokemon.append(pokemon)
+                    #remove as egg and add as pokemon
+                    inventory.pokemons().remove(pokemon['id'])
+                    inventory.pokemons().add(inventory.Pokemon(pokemon))
```

Comment:
It seems like it is going to work like that. It is hard to be entirely sure since this file is a maze.
The `_check_inventory` should not call the api (at every loop !) so, removing the egg and adding the Pokemon will have to be moved in the `_hatch_eggs` function one day or another.